### PR TITLE
404 page

### PR DESCRIPTION
**Pull request by gnocchi2815**
I've created a basic but improved 404 page, in response to issue #106. Perhaps someone would like to improve on this further by adding a 404 related tweet as suggested in the issue.

### CODE REVIEW

File: dwitter/templates/404.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+    <h2 class=header-title>This dweet doesn't exist.</h2>
```
Comment (lionleaf) [inline dwitter/templates/404.html]: You'd get a 404 for other cases than missing dweets. Maybe change to "this page doesn't exist"

Comment (gnocchi2815) [inline dwitter/templates/404.html]: Thanks for the feedback - good point!

Comment (gnocchi2815) [inline dwitter/templates/404.html]: Updated

**Comment by lionleaf** [comment]
Thanks! Deployed: https://www.dwitter.net/lalalalakwd :)

**Comment by lionleaf** [comment]
Currently had to disable it for a while. It was used for dweet.dwitter.net as well (where I host the dweet content) and someone wrote a dweet that references a random one. Because dweet.dwitter.net doesn't have the same auth session it would crash and flood my inbox. Will restore the 404 once I have time to fix that

**Pull request merged**